tests/data/test.properties

```diff
@@ -0,0 +1,13 @@
+# Any copyright is dedicated to the Public Domain.
+# http://creativecommons.org/publicdomain/zero/1.0/
+1=1
+ 2=2
+3 =3
+ 4 =4
+5=5
+6= 6
```

Comment:
Remembered this one:
https://github.com/mozilla/pontoon/issues/2058

What do you think should be the next step for that issue?

Reply:
With this library, leading and trailing whitespace in .properties values is properly escaped: https://github.com/mozilla/moz-l10n/blob/4acc215109e02b560f8a0a1d031db1689e522b84/moz/l10n/properties/serialize.py#L93-L96